osmaxx-py/osmaxx/utilities/shortcuts.py

```diff
@@ -32,3 +32,16 @@ def get_list_or_none(klass, *args, **kwargs):
     if not object_list:
         return None
     return object_list
+
+
+def get_actual(obj):
+    """Expands `obj` to the actual object type.
```

Comment:
Documentation of this function should probably work with inheritance-related terms like 'parent type' / 'child type' or 'super type' / 'subtype', or if this works for indirect inheritance 'ancestor type' / 'descendent type'.